config/routes.rb

```diff
@@ -25,6 +25,10 @@
   get "/tour", to: "help#tour"
   get "/help/cookies", to: "help#cookie_settings"
 
+  # Performance platform pages
+  get "/performance" => "performance#index"
+  get "/performance/*other" => "performance#redirect_to_index"
```

Comment:
An alternative to this might be to publish two special routes. One exact route to`/performance` and one prefix route to `/performance` which redirects to `/performance`. This means the redirect will be handled by router, so wouldn't need to go so far into our stack, but I'm not sure how router would handle two routes at `/performance` one exact and one prefix. In some ways I like that the logic is contained in this app rather than "hidden" away in router, so I don't really mind which way we decide to go, just wanted to present it as an option.

Reply:
Thanks - hadn't thought of that. I like the idea of it being slightly more efficient, but code cohesion probably better here 👍